src/cpu/aarch64.cpp

```diff
@@ -232,6 +239,7 @@ void megpeak::aarch64() {
     benchmark(mul_throughput, mul_latency, "mul");
 #if defined(__ARM_FEATURE_MATMUL_INT8)
     benchmark(smmla_throughput, smmla_latency, "smmla", 64);
+    benchmark(bfmmla_throughput, bfmmla_latency, "bfmmla", 64);
```

Comment:
the operators of bfmmla is 64?

Reply:
it has 2x4x2 multiplication，and 2x4x2 addition，so it should be 32

Reply:
yes should be 32

Reply:
fix it now